### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: The latest version of Laravel has a dependency for `phpunit/phpunit` of `^9.3`. If you would like to also upgrade your tests, you may run the [PHPUnit 9 Shift](https://laravelshift.com/upgrade-phpunit-9) for free.

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/calebporzio/parental/pull/101/commits/722c7ab365b8315a3b0243130f56d88af3a74c11?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/parental.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "calebporzio/parental": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -9,12 +9,12 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.4, 8.0]
-        laravel: [8.*]
+        php: [8.0]
+        laravel: [9.*]
         dependency-version: [prefer-stable]
         include:
```
Comment (laravel-shift) [inline .github/workflows/tests.yml:15]: This `include` is probably unnecessary given it matches the current constraints.

**Comment by JustSteveKing** [comment]
Waiting for stable 9 to be released so tests can pass 😃

**Comment by jasonmccreary** [comment]
@JustSteveKing, if you want, you can configure the min stability options in `composer.json` just to see if the tests pass.

**Comment by JustSteveKing** [comment]
Tested locally - works wonderfully @jasonmccreary 🔥

**Comment by miken32** [comment]
Will a release be pushed out soon so this shows up in packagist?

**Comment by JustSteveKing** [comment]
@miken32 released!

https://github.com/calebporzio/parental/releases/tag/v1.1.0

**Pull request merged**